.dockerignore

```diff
@@ -2,6 +2,7 @@
 *
 
 # Explicitly whitelist files relevant to image creation.
+!deployment
```

Comment:
Is `deployment` now being whitelisted because it's the source of the Hasura metadata/Postgres SQL files?

Reply:
Yes exactly